tests/artifactorylocalrepository_test.go

```diff
@@ -31,7 +32,7 @@ func TestArtifactoryLocalRepository(t *testing.T) {
 	t.Run("localOkgTest", localOpkgTest)
 	t.Run("localComposerTest", localComposerTest)
 	t.Run("localvagrantTest", localVagrantTest)
-	t.Run("localGenericTest", localGenericTest)
```

Comment:
I'm guessing that *localGenericTest* was removed by mistake.

Reply:
Yes, thanks!.